api/opensource/mfx_dispatch/src/mfx_plugin_hive_linux.cpp

```diff
@@ -40,7 +40,9 @@
 
 namespace 
 {
-    const char rootPluginPath[] = "/opt/intel/mediasdk/plugins/plugins.cfg";
+    const char *rootPluginPath[2] = { getenv("MFX_PLUGINS_PATH" ),
+                                      "/opt/intel/mediasdk/plugins/plugins.cfg"
+                                    };
```

Comment:
Can you make this configurable at compile time, too?  For example, add -DMFX_PLUGINS_DIR=... to compiler and use `MFX_PLUGINS_DIR "/plugins.cfg"` here instead of hardcoded /opt/intel/mediasdk/plugins/plugins.cfg.  In cmake files, MFX_PLUGINS_DIR is set to `$CMAKE_INSTALL_PREFIX/plugins` and `CMAKE_INSTALL_PREFIX=/opt/intel/mediasdk` by default.  LibVA uses a similar technique for the VA_DRIVERS_PATH.

Reply:
Good idea, will update this patch. : )